API/Api/Api.ts

```diff
@@ -19,12 +21,17 @@ class WhiteToothAPI {
         app.get('*', (req: express.Request, res: express.Response) => {
             try {
                 const parsedURL = this.parseRequestURL(req.url);
-                const environment = JSON.parse(this._environmentRepository.getEnvironmentByName(parsedURL.environment));
-                const response = environment.responses[parsedURL.url];
+                const environment = this._environmentRepository.getEnvironmentByName(parsedURL.environment);
+
+                if (_.isEmpty(environment) || _.isEmpty(environment.responses))
```

Comment:
is this possible to have env without responses?
I remember we discussed about default responses

Reply:
we can enforce it in the constructor if we get all environments and validate it - but I don't think it's efficient